src/components/shared/Accordion.css

```diff
@@ -15,18 +15,22 @@
 .accordion {
   background-color: var(--theme-sidebar-background);
   width: 100%;
+  list-style-type: none;
+  padding-left: 0px;
+  margin-top: 0px;
 }
 
 .accordion ._header {
   background-color: var(--accordion-header-background);
   border-bottom: 1px solid var(--theme-splitter-color);
   display: flex;
   font-size: 12px;
-  padding: 4px;
+  padding: 0px;
```

Comment:
We should keep padding to the left (for LTR) and right (for RTL);  something like:

```css
.accordion ._header {
  padding: 0 4px;
}
```